### PR TITLE
Display the popover Source/Destination info side-by-side

### LINKED ISSUE #1661: Display Src/Dst info in popover side by side
Currently, the source/destination popover displays Source above Destination, but I think it'd be more helpful if it were displayed side by side e.g.
```
SOURCE            DESTINATION
deploy/books      deploy/authors
pod/books-3kk2l3  pod/authors-2lk34
ip/10.1.0.1       ip/10.0.1.0
```

Current state:
screen shot 2018-09-17 at 10 17 16 am

### PR DESCRIPTION
**Pull request by rmars**
Previously, we would display source and destination info in the Top/Tap table popovers in a vertical format. This PR places them in a table so that each type of source/dest (ip, pod, pod owner) can be read left to right.

screen shot 2018-09-24 at 12 58 18 pm
screen shot 2018-09-24 at 12 58 05 pm
screen shot 2018-09-24 at 12 57 58 pm

Fixes #1661

### CODE REVIEW

File: web/app/js/components/util/TapUtils.jsx [hunk 1]
```diff
@@ -163,52 +163,88 @@ const publicAddressToString = ipv4 => {
...
 
-// display consists of a list of ips and pods for aggregated displays (Top)
-const displayLimit = 3;
-const resourceSection = (endpoint, display, labels, ResourceLink) => {
-  let ipList = null;
-  let podList = null;
+const displayLimit = 3; // how many upstreams/downstreams to display in the popover table
+const popoverSrcDstColumns = [
+  { title: "Source", dataIndex: "source", key: "source" },
+  { title: "Destination", dataIndex: "destination", key: "destination" }
```
Comment (klingerf) [inline web/app/js/components/util/TapUtils.jsx]: Would it be possible to add another column between Source and Destination, with some sort of icon that indicates the direction of the traffic? It's a bit hard to parse visually that traffic is flowing from column 1 to column 2. For instance:

image

In this example it's hard to tell that there are two columns. I think maybe adding something like `10.1.01 ➡️ 10.1.14.247` would make it clear.

Comment (rmars) [reply web/app/js/components/util/TapUtils.jsx]: ooooh, interesting. sure!

Comment (rmars) [reply web/app/js/components/util/TapUtils.jsx]: screen shot 2018-09-24 at 5 17 55 pm

Comment (klingerf) [reply web/app/js/components/util/TapUtils.jsx]: Thanks! At the risk of bike-shedding this wayyy too much, how about:

```
  { title: "", render: () => <i className="fas fa-long-arrow-alt-right" />},
```

Comment (rmars) [reply web/app/js/components/util/TapUtils.jsx]: haha sure!
